plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -159,12 +159,17 @@ class NewCreateServiceModalForm extends Component {
       return true;
     }
 
+    const isBaseConfigChanged = this.state.baseConfig !== nextState.baseConfig;
```

Comment:
I am actually unsure how you want it, so I left it for you to do. Or, could you be a bit more specific, so I know what to do here? Do you want _every_ variable in this change renamed to `did`?